labs-ios-starter/Profiles/Model Controllers/ProfileController.swift

```diff
@@ -246,7 +246,9 @@ class ProfileController {
         let requestURL = baseURL.appendingPathComponent("profiles")
         guard var request = networkService.createRequest(url: requestURL, method: .post) else {
             print("invalid request")
```

Comment:
```suggestion
            print("Failed to create request")
```

Reply:
Semantics are good, and this looks like something I did in a rush, but :

"invalid request" 
vs 
"failed to create a valid request" 

doesn't change a lot to tell us where the error occurred since this method is used in multiple places. How about if I change it to: "Failed to create a valid request when fetching profiles"?